spacy/scorer.py

```diff
@@ -20,10 +20,16 @@
 class PRFScore:
     """A precision / recall / F score."""
 
-    def __init__(self) -> None:
-        self.tp = 0
-        self.fp = 0
-        self.fn = 0
+    def __init__(
+        self,
+        *,
+        tp: Optional[int] = None,
+        fp: Optional[int] = None,
+        fn: Optional[int] = None,
+    ) -> None:
+        self.tp = 0 if tp is None else tp
+        self.fp = 0 if fp is None else fp
+        self.fn = 0 if fn is None else fn
```

Comment:
Good catch, and thanks for the fix!

Perhaps we could simplify further?

```suggestion
    def __init__(
        self,
        *,
        tp: Optional[int] = 0,
        fp: Optional[int] = 0,
        fn: Optional[int] = 0,
    ) -> None:
        self.tp = tp
        self.fp = fp
        self.fn = fn
```